rnote-ui/src/canvas/imexport.rs

```diff
@@ -302,15 +302,25 @@ impl RnCanvas {
 
         if let Err(e) = res {
             self.set_save_in_progress(false);
+            if self.recovery_in_progress() {
+                self.set_recovery_in_progress(false)
+            }
 
             // If the file operations failed in any way, we make sure to clear the expect_write flag
             // because we can't know for sure if the output_file monitor will be able to.
```

Comment:
@flxzt How important if this, when the user isn't supposed to access the folder where the file gets saved (as in the case of recovery files). Do I need to make another monitor for the recovery file?

Reply:
Sorry I didn't look at the code too closely so far since this you marked it as a draft, I don't really know